model/src/lib.rs

```diff
@@ -458,6 +458,7 @@ impl From<Decimal> for Percent {
 }
 
 #[derive(Debug, Clone, Copy, Serialize, Deserialize, PartialEq, Eq, Hash, EnumIter, Display)]
+#[strum(serialize_all = "UPPERCASE")]
```

Comment:
🔧 I know it's a bit annoying but it would be good to have a dedicated type (or simply some specific serialisation logic) in `position_metric` instead of this.

I can see my future self changing this and breaking the metrics by mistake.

Reply:
ay, indeed, model shouldn't change because of projection needs :p

a `ContractSymbol` in projection would also allow me to write `From` trait in my latest PR, so it's not the first time it might have been handy.

Reply:
added a new small issue, I'll put in on my plate for today.